src/transport_performance/utils/raster.py

```diff
@@ -73,6 +77,7 @@ def merge_raster_files(
 
     """
     # defend against case where the provided input dir does not exist
+    # note: not able to use `_check_parent_dir_exists()` here since its a dir
```

Comment:
It is mentioned here that `_check_parent_dir_exists()` can not be used. In theory, a dummy file name e.g., `dummy.txt` can be appended onto `input_dir` to include this functionality.  
We would need to have `create=False` in this instance as we only want to know if the directory exists.  
While this isn't a required change, it would improve consistency across all directory checks.

Reply:
Yes, this could work - great idea! I'll make it very clear in a comment the origin of the dummy file, so that we don't forget going forward.